backend/api/Controllers/Models/ScheduledMissionQuery.cs

```diff
@@ -4,7 +4,6 @@ public struct ScheduledMissionQuery
     {
         public string RobotId { get; set; }
         public int EchoMissionId { get; set; }
-        public DateTimeOffset StartTime { get; set; }
```

Comment:
Why are we removing this option? 
If we do, we should rename from 'Schedule' to 'Queue'

Reply:
Even if we want to remove the option for frontend users, we don't have to remove it as an option in the backend

Reply:
I reintroduced startTime but set it as optional